channels/cliprdr/cliprdr_common.c

```diff
@@ -299,9 +299,9 @@ wStream* cliprdr_packet_format_list_new(const CLIPRDR_FORMAT_LIST* formatList,
 						Stream_Free(s, TRUE);
 						return NULL;
 					}
-					formatNameSize += 1; /* append terminating '\0' */
 				}
 
+				formatNameSize++;
```

Comment:
Why moving this out of the upper `if`-block? This is the ASCII format block (short format names), where the name always uses 32 bytes (unused chars are zeros). If `szFormatName == NULL`, then now only 30 zeros are written, while before 32 where written.

Reply:
because the remaining 2 zero bytes are written down below in line 372